memtar.gemspec

```diff
@@ -12,14 +12,16 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/conjurinc/memtar"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files          = `git ls-files`.split($\)
```

Comment:
Formatting is slightly off if you want to fix during a rebase.

Reply:
Fixed!